lib/mix/lib/mix/tasks/release.ex

```diff
@@ -622,6 +622,13 @@ defmodule Mix.Tasks.Release do
       files to. It can be set to a custom directory. It defaults to
       `$RELEASE_ROOT/tmp`
 
+    * `RELEASE_DISTRIBUTION` - how do we want to run the distribution.
+      Using `name` (long names) or `sname` (short names). Defaults to
+      `sname` which allows access only within the current system.
+      `name` allows external connections. If `name` is used and you are
+      running on a version earlier than Erlang/OTP 22+, you must set
```

Comment:
Nitpick, but I think either we say "on a version earlier than Erlang/OTP 22", or we say "on Erlang/OTP 22+" (my preference is the former).